Service/LeantimeHelper.php

```diff
@@ -119,6 +135,22 @@ private function post(string $method, array $params = []): ResponseInterface|str
         } catch (Exception $e) {
             \Helper::logException($e);
 
+            $counter = [
+              'name_space' => 'leantime_helper',
+              'name' => 'post_exception',
+              'help' => 'Increases when posting to leantime fails'
+            ];
+
+            $labels = [
+              'module' => 'itk_issue_create',
+              'service' => 'LeantimeHelper',
+              'method' => 'post',
+              'exception' => $e->getMessage(),
+              'code' => $e->getCode(),
+            ];
```

Comment:
You should/can not have exception message and code as labels... that would create way to many vectors/indexes in Prometheus and could make it go down over time (memory usage would explode). Labels should be stuff that do change over time.

e.g
type => "exception"
method => "post "

I would drop service "LeantimeHelper" also as you already have that as part of the name (here call namespace).